app/src/main/java/com/duckduckgo/app/global/shortcut/AppShortcutCreator.kt

```diff
@@ -125,9 +134,27 @@ class AppShortcutCreator @Inject constructor(
             .build().toShortcutInfo()
     }
 
+    private fun buildAndroidDesignSystemShortcut(context: Context): ShortcutInfo {
+        val browserActivity = BrowserActivity.intent(context).also { it.action = Intent.ACTION_VIEW }
+        val settingsActivity = SettingsActivity.intent(context).also { it.action = Intent.ACTION_VIEW }
+        val adsActivity = AppComponentsActivity.intent(context).also { it.action = Intent.ACTION_VIEW }
+
+        val stackBuilder = TaskStackBuilder.create(context)
+            .addNextIntent(browserActivity)
+            .addNextIntent(settingsActivity)
+            .addNextIntent(adsActivity)
+
+        return ShortcutInfoCompat.Builder(context, SHORTCUT_ID_ADS_DEMO)
+            .setShortLabel(context.getString(com.duckduckgo.mobile.android.R.string.ads_demo_activity_title))
+            .setIcon(IconCompat.createWithResource(context, com.duckduckgo.mobile.android.R.drawable.ic_dax_icon))
+            .setIntents(stackBuilder.intents)
+            .build().toShortcutInfo()
+    }
+
     companion object {
         private const val SHORTCUT_ID_CLEAR_DATA = "clearData"
         private const val SHORTCUT_ID_NEW_TAB = "newTab"
         private const val SHORTCUT_ID_SHOW_BOOKMARKS = "showBookmarks"
+        private const val SHORTCUT_ID_ADS_DEMO = "adsDemo"
```

Comment:
nit: "adsDemo" reads like this is an ads demo (as in, advertisements) rather than ADS

Reply:
good catch, changing it now.